src/lib/components/buttons/IconButton.svelte

```diff
@@ -1,7 +1,28 @@
 <script lang="ts">
-    export let text: string;
-    export let onClick;
+  export let classProp: string = "";
+  export let onClick: (e) => void;
+  export let disabled: boolean = false;
+  export let width: number = undefined;
```

Comment:
We are declaring the variable as number type, yet assigning it undefined. This might cause problems.